### DISCUSSION TITLE
search for strings by date and history

### DISCUSSION THREAD

**Discussion by PawelSpoon** [WeblateOrg/weblate#4851] [Q&A]
Hi,
due to some configuration settings changes, i do have now 1000+ strings in translated state, that were previously in approved state.
i know when this change did happen (the day)
i can see in the history that they were approved before.
so can i somehow search for that combination ?
ideally i can set them back automated, but to have a stupid long list, that i need to click 1000 time would already help
Thanks a lot !

**Comment by nijel** [comment]
`changed:2020-11-05` should find all strings changed in that date, but you cannot search for what has happened back then. Maybe `changed:2020-11-05 AND state:trasnlated` is good enough for you? Adding `changed_by:username` might also help.

See docs on searching for more details: https://docs.weblate.org/en/latest/user/search.html

**Reply by PawelSpoon** [reply]
that is what i am using but far from perfect. cause it returns all the objects that are now not approved and were changed on that day.
60% of the hits was previously in approved state, 30% were only translated.
but after 8hrs of checking every single translation modified on last monday i am almost done

**Reply by nijel** [reply]
Weblate 4.4 will have support for more detailed searching in history, see https://github.com/WeblateOrg/weblate/commit/47f011a6f049b2c41d29a6ce4071591d9207f0b5 and https://docs.weblate.org/en/latest/user/search.html#search-changes (it will take some time till the docs is online).
